core/balances.go

```diff
@@ -566,6 +584,10 @@ func (bc *BaseContract) IndustrialBalanceUnLock(
 	address *types.Address,
 	amount *big.Int,
 ) error {
+	if stub, ok := bc.GetStub().(*cachestub.TxCacheStub); ok {
+		stub.AddAccountingRecord(bc.config.GetSymbol(), address, address, amount, "industrial balance unlock")
```

Comment:
should be changed to something similar to `symbol_barId`
look at working with idustrial